core/web/resolver/query.go

```diff
@@ -308,3 +309,39 @@ func (r *Resolver) JobProposal(ctx context.Context, args struct {
 
 	return NewJobProposalPayload(jp, err), nil
 }
+
+// Nodes retrieves a paginated list of nodes.
+func (r *Resolver) Nodes(ctx context.Context, args struct {
+	ChainID *graphql.ID
```

Comment:
We don't this `ChainID` filter because you can get the nodes for a chain via the chain query